index.html

```diff
@@ -10,13 +10,30 @@
 </head>
 <body>
 
+  <section class="entire">
+  <section >
+    
+  
+  </section>
+
 <section class="container">
+    <div class="menu-popup hidden" id="popup"> 
+      <div onclick="popmenu()" class="close-mobile-menu" id="close-mobile-menu">
+        <img src="./Icons/Icon - Cancel.png" alt="close menu button"/>
+      </div>   
+    <div class="menu-li">
+        <nav><a href="#Portfolio" onclick="popmenu()">Portfolio</a></nav>

```

Comment:
![image](https://user-images.githubusercontent.com/93039359/224921529-65291d82-0270-4682-ba6e-d45bff3e5c16.png)

**[OPTIONAL]**


- [ ] KIndly fix this button because it is not taking us to the right section of your page 😨 It is taking us to your homepage but it should take us to the cards which you have by the class name of **middle-section1 & middle-section2** 😊 Kindly fix this issue to make your project professional 👍